src/Model/Filter/Boolean.php

```diff
@@ -25,7 +25,12 @@ public function process()
             return;
         }
 
-        $value = strtolower($this->value());
+        $value = $this->value();
+        if (!is_scalar($value)) {
+            return;
+        }
+
+        $value = strtolower($value);
```

Comment:
The default `falsy` lists contain boolean `false` too and `strtolower(false)` will return an empty string.